_benchmark/user-guide/concepts.md

```diff
@@ -25,6 +25,93 @@ A workload is a specification of one or more benchmarking scenarios. A workload
+
+OSB makes function calls to `opensearch-py`, a client for OpenSearch, to communicate with an OpenSearch cluster. OSB tracks the time between when the `opensearch-py` client sends the request and receives a response from the OpenSearch cluster and considers this to be the service time. Contrasting with the traditional definition of service time, OSB’s definition of service time includes overhead, such as network latency, load-balancer overhead, or deserialization/serialization. The following image illustrates how OpenSearch measure service time versus the traditional measurement. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/benchmark/service-time.png" alt="">
+
+### Latency
+
+Target throughput is core to understanding OSB’s definition of **latency**. Target throughput is the rate at which OSB issues requests  and assumes the responses will come back instantaneously.  `target-throughput` is one of the common workload parameters that can be set for each test and is measured in operations per second.
+
+OSB always issue requests one at a time for a single client thread, specified as `search-clients` in workload parameters. If `target-throughput` is set to `0`, OSB issues a request immediately after it receives the response from the previous request. If the `target-throughput` is not 0, OSB issues the next request to match the `target-throughput` set, assuming responses return instantaneously.
+
+#### Example A
+
+The following diagrams illustrate how latency is calculated when we expect a request response time of 200ms and use the following settings: 
+
+- `search-clients` is set to `1` 
+- `target-throughput` is set to `1` operations per second. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-1.png" alt="">
+
+However, when a request takes longer than 200ms, such as when a request takes 1110ms instead of 400ms, OSB sends the next request that was supposed to occur at 4.00s based on the `target-throughput` at 4.10s. All subsequent requests after the 4.10s request attempt to resynchronize with the `target-throughput` setting.
+
+<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-2.png" alt="">
+
+When measuring the overall latency, OSB looks all the requests performed. All requests have a latency of 200ms except for the following two: 
+- The request that lasted 1100ms 
+- The following request which was supposed to start at 4:00s. This request was delayed by 100ms, denoted by the orange area in the following diagram, and had a response of 200ms. 
+
+
+<img src="{{site.url}}{{site.baseurl}}/images/benchmark/latency-explanation-3.png" alt="">
+
+Therefore, the overall latency OSB calculates in this scenario, when accounting for the delayed response time with the response time, is **300ms**.
```

Comment:
A couple of things: 
1. Could we move this sentence up and join line 75 so that readers know that it relates to this request and not all requests. 

2. Could we rephrase this line from `the overall latency OSB calculates in this scenario, when accounting for the delayed response time with the response time, is **300ms**.` to `When calculating the latency for this request, OSB will account for the delayed start time and combine it with the response time. Thus, the latency for this request is **300ms**.`

Reply:
Is the latency measurement specific to the request or the average latency for all requests @IanHoang?

Reply:
it's specific to the request